<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLogin</class>
 <widget class="QMainWindow" name="frmLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmLogin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelInformed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Log In or Create an Account</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxLogin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title">
      <string>Login</string>
     </property>
     <widget class="QLabel" name="labelUserName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsername">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPassword">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/login.png</normaloff>images/login.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/cancel.png</normaloff>images/cancel.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameUserType">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select user to login</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbUsers">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Admin</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/admin-ui.png</normaloff>images/admin-ui.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Professor</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/teacher.png</normaloff>images/teacher.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Student</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/student.png</normaloff>images/student.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Staff</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/staff.png</normaloff>images/staff.png</iconset>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
